Makefile

```
@@ -16,12 +21,13 @@ messages:
 locale:
 	uv run manage.py compilemessages -l ru
 
-static:
+static: frontend
 	uv run manage.py collectstatic --no-input
 
-deploy:
+deploy: frontend
```

checkmake (0.3.2)

[warning] Target body for "deploy" exceeds allowed length of 5 lines (11).

(maxbodylength)